scripts/notarize.js

```diff
@@ -9,7 +9,8 @@ const config = require('../electron-builder.json');
 
 exports.default = async function notarizing(context) {
     const {electronPlatformName, appOutDir} = context;
-    if (electronPlatformName !== 'darwin' || process.platform !== 'darwin') {
+
+    if (!(['darwin', 'mas'].includes(electronPlatformName) && process.platform === 'darwin')) {
```

Comment:
Is this right? Does mas need notarization? Is there a benefit to notarizing the mas build?

Reply:
We originally thought it did but it actually doesn't. I'll revert this change.